content/platforms/kubernetes/pod-scheduling.md

```diff
@@ -181,38 +168,31 @@ spec:
       topologyKey: kubernetes.io/hostname
 ```
 
-If other database's pods are deployed with the label `local/role: database`,
-the Redis Enterprise pods will also not be co-located on the same node.
+In this case, any pods that are deployed with the label `local/role: database` cannot be scheduled on the same node.
 
 
 ## Using rack awareness
 
-Redis Enterprise can be configured to be rack aware so that the availability
-is increased in the event of partitions or other rack (or region) related
-failures.
+You can configure Redis Enterprise with rack-zone awareness to increase availability
+during partitions or other rack (or region) related failures.
 
-Rack awareness is a single property in the Redis Enterprise cluster CRD named
-`rackAwarenessNodeLabel` whose value is the rack label applied to the node. This
-label is often `failure-domain.beta.kubernetes.io/zone` as documented in
+Rack-zone awareness is a single property in the Redis Enterprise cluster CRD named `rackAwarenessNodeLabel`.
+This value for this label is commonly `failure-domain.beta.kubernetes.io/zone` as documented in
 ['Running in multiple zones'](https://kubernetes.io/docs/setup/best-practices/multiple-zones/#nodes-are-labeled).
 
-You can check your nodes are labeled via:
+You can check the value for this label in your nodes with the command:
 
 ```sh
-kubectl get nodes -o custom-columns="name:metadata.name","rack\\zone:metadata.labels.failure-domain\.beta\.kubernetes\.io/zone"
-```
+$kubectl get nodes -o custom-columns="name:metadata.name","rack\\zone:metadata.labels.failure-domain\.beta\.kubernetes\.io/zone"
 
-The output might be something like:
-
-```sh
 name                                            rack\zone
 ip-10-0-x-a.eu-central-1.compute.internal    eu-central-1a
 ip-10-0-x-b.eu-central-1.compute.internal    eu-central-1a
 ip-10-0-x-c.eu-central-1.compute.internal    eu-central-1b
 ip-10-0-x-d.eu-central-1.compute.internal    eu-central-1b
 ```
 
-All you need to do is tell the operator which label represents the rack zone:
+You can configure the CRD to use another value to represent the rack-zone:
```

Comment:
You must specify a value. This is not specified by default.

"You can configure the node label to read for the rack zone by setting the `rackAwarenessNodeLabel` property:"